amazon_msk/metadata.csv

```diff
@@ -338,31 +341,60 @@ aws.msk.process.virtual.memory.max.bytes,gauge,,byte,,Maximum amount of virtual
 aws.msk.promhttp.metric.handler.requests.in.flight,gauge,,,,Current number of scrapes being served.,0,amazon_kafka,,
 aws.msk.promhttp.metric.handler.requests.total,count,,,,Total number of scrapes by HTTP status code.,0,amazon_kafka,,
 aws.msk.kafka.network.request.ErrorsPerSec,gauge,,error,,,0,amazon_kafka,,
+aws.msk.kafka.network.request.ErrorsPerSec.count,count,,error,,,0,amazon_kafka,,
```

Comment:
Small nit, but I'm wondering if we can add descriptions for these metrics? I know these are metrics from labels and they don't have descriptions from Prometheus, but I found [this Kafka doc](https://kafka.apache.org/11/documentation.html#monitoring) which should have the corresponding metrics. This is mostly just so we can add `... as a count` at the end of the description to help users differentiate between them.